docs/standard/microservices-architecture/architect-microservice-container-applications/index.md

```diff
@@ -20,7 +20,7 @@ Enterprise applications can be complex and are often composed of multiple servic
 
 In the container model, a container image instance represents a single process. By defining a container image as a process boundary, you can create primitives that can be used to scale the process or to batch it.
 
-When you design a container image, you will see an [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/) definition in the Dockerfile. This defines the process whose lifetime controls the lifetime of the container. When the process completes, the container lifecycle ends. Containers might represent long-running processes like web servers, but can also represent short-lived processes like batch jobs, which formerly might have been implemented as Azure [WebJobs](https://azure.microsoft.com/en-us/documentation/articles/websites-webjobs-resources/).
+When you design a container image, you will see an [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/) definition in the Dockerfile. This defines the process whose lifetime controls the lifetime of the container. When the process completes, the container lifecycle ends. Containers might represent long-running processes like web servers, but can also represent short-lived processes like batch jobs, which formerly might have been implemented as Azure [WebJobs](https://azure.microsoft.com/documentation/articles/websites-webjobs-resources/).
```

Comment:
change https://azure.microsoft.com/documentation/articles/websites-webjobs-resources/ to https://docs.microsoft.com/azure/app-service-web/websites-webjobs-resources